uniffi_bindgen/src/interface/mod.rs

```diff
@@ -336,13 +323,21 @@ impl APIBuilder for weedle::Definition<'_> {
                     false
                 };
                 if is_error {
-                    ci.add_error_definition(d.convert(ci)?)
+                    let err = d.convert(ci)?;
+                    ci.add_error_definition(err)
```

Comment:
This and similar changes, is because `convert()` and friends now take an `&mut ci`, and Rust's order of evaluation tries to take the mutable reference for `ci.add_error_definition` before evaluating any of its arguments. So it thinks I'm trying to do a double-mutable-borrow unless I pull the conversion into a separate line.